index.js

```diff
@@ -97,11 +97,16 @@ function Bankai (entry, opts) {
   })
 
   // Insert nodes into the graph.
+  var documentsDepList = [ 'assets:list', 'manifest:bundle', 'styles:bundle', 'scripts:bundle' ]
+
+  if (opts.watch !== false) {
```

Comment:
I think the different `opts.watch` / `metadata.reload` / `metadata.watch` checks here and below are a little confusing. Maybe it would be easier to understand if, at the start of this function, we default `opts.reload` to the value of `opts.watch` if it is not set? Then we could check `if (opts.reload)` here and `if (state.metadata.reload)` below  in graph-document.js.

Reply:
Yes

Reply:
I agree to add an `opts.reload`, But I think it can't check totally below in graph-document.js, because document node doesn't dependent on reload node when on `bankai build` run.

Reply:
if we set `opts.reload` inside the Bankai constructor in index.js it should work I think? then you can use it in the condition on this line before adding the `reload` node. and it will be available inside the document node as `metadata.reload`. (we already pass `reload: Boolean(opts.reload)`) Or am I missing something? :thinking:

Reply:
You are right, I will create a feature branch for this.